docs/attributes-registry/user.md

```diff
@@ -0,0 +1,20 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# User
+
+## User Attributes
+
+<!-- semconv registry.user(omit_requirement_level) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `user.domain` | string | Name of the directory the user is a member of. For example, an LDAP or Active Directory domain name. | `internal.example.com` |
+| `user.email` | string | User email address. | `a.einstein@example.com` |
+| `user.full_name` | string | User's full name | `Albert Einstein` |
```

Comment:
We were going to add PII or potential PII annotations to semconv registry at some point.  This is a non-blocking comment reminding me to prioritize that work.

Reply:
This really requires some support for additional "metadata" for the value of the semantic convention so that people that "pass" PII can tag it as containing PII vs people that just pass a standard string. 
This would have been a usage / reason for why "nested" attributes could have been used for. Without that is would need to be either JSON encoded or contain some form of encoding.

Reply:
I was thinking about additional `tag` added to the specific attribute as we do now with `stability` for example

Reply:
It'd be great to add a way to specify that certain attribute contain sensitive information.

The trick we usually do with things like that:
- tell that instrumentations can scrub sensitive data if they can. E.g. here https://github.com/open-telemetry/semantic-conventions/pull/676
- Make them opt-in (if they are likely to contain sensitive info)

If we had a common way, we would prevent things like this https://github.com/open-telemetry/semantic-conventions/issues/626

It'd be nice to add a section in the registry  similar to one we have under URL - https://github.com/open-telemetry/semantic-conventions/blob/405e120833081810b81f1b5dbe1467bc64fa3bc2/docs/url/url.md#sensitive-information.

We also have unofficial-undocumented precedent of marking things as sensitive in yaml here: https://github.com/open-telemetry/semantic-conventions/blob/19f2ba7d0c0b0adc1b9efb8ab189c5300e038e4d/model/url.yaml#L9

Reply:
Yes, I agree that we should have a tag/section for sensitive properties in registry as it should be the source of truth for all namespaces

Reply:
It's a little more that just tagging the telemetry in semconv registry (although that is a good start), some customers / applications are quite happy to pass un-obfuscated information into the fields (think corporate controlled environments) while others would want the standard obfuscation (hashing etc) applied by an instrumentation or receiver before storage.

To be a little more explicit
- A Corporate application that runs in a corporate control environment may want to log the full unmasked username for all events for full audit tracking. (And yes this does occur)
- Generic application that runs on end-user machines (uncontrol environments) would want to conform with EUDB / PII requirements and thus have all PII data "handled" (whether that be before leaving the client machine or on the receiver side before storing)

The ways this is typically handled is either
- The "value" passed in contains the "tags" indicating that they want the data obfuscated (or not)
- There is another parallel set of "meta-data" indicating the tags that should be applied to different fields (not ideal)

From a semconv "flat" attribute perspective this would mean either
- `semconv.object.fieldname` = "sensitive value"
- `semconv.object.fieldname.metadata` = &lt;tag - to identify how to handle>
- or `metadata.semconv.object.fieldname` = &lt;tag - to identify how to handle>

or the metadata is encoded within the value (possible examples only - others obviously exist)
- `semconv.object.fieldname` = "&lt;tag>:sensitive value"
- `semconv.object.fieldname` = '{ "tag": &lt;tag - to identify how to handle>, "value": ":sensitive value"}'

OpenTelemetry "should" define a standard way to perform this operation so that all receivers (and senders) understand the implications.

Reply:
hey @MSNev this is great addition. And we should definitely decide how to and implement it 
Here I refer explicitly to schema and definition there. Sensitive tag here is first step in that direction and changes in instrumentation should be a follow-up